go-controller/pkg/ovn/gateway_init.go

```diff
@@ -146,8 +146,12 @@ func (oc *DefaultNetworkController) gatewayInit(nodeName string, clusterIPSubnet
 		Copp:        &oc.defaultCOPPUUID,
 	}
 
-	if oc.clusterLoadBalancerGroupUUID != "" && oc.routerLoadBalancerGroupUUID != "" {
-		logicalRouter.LoadBalancerGroup = []string{oc.clusterLoadBalancerGroupUUID, oc.routerLoadBalancerGroupUUID}
+	if oc.clusterLoadBalancerGroupUUID != "" {
+		logicalRouter.LoadBalancerGroup = []string{oc.clusterLoadBalancerGroupUUID}
+		if l3GatewayConfig.NodePortEnable && oc.routerLoadBalancerGroupUUID != "" {
+			// add ClusterRouterLBGroupName to the gateway router only if nodePort is enabled
```

Comment:
nit: this comment should be add routerLoadBalancerGroupUUID right?

Reply:
thanks. updated.